moredata/utils/osm_downloader.py

```diff
@@ -53,10 +53,11 @@ def get_places(self, place_name, key, value, query=None, tags=("name","geom")):
             properties = data["properties"]
             if "tags" in properties.keys():
                 tags = properties["tags"]
-
                 for item in tags.items():
-                    key, value = item[0], item[1]
-                    properties[key] = value
+                    k, v = item[0], item[1]
+                    properties[k]=v
+                    properties['value'] = v
```

Comment:
@thomassonobe This and the next LOC would be outside the loop. Besides, instead `properties['value']=v` and `properties['key']=k`, use `properties['value']=**value**` and `properties['key']=**key**`